packages/community-cli-plugin/src/commands/bundle/buildBundle.js

```diff
@@ -18,6 +18,7 @@ import parseKeyValueParamArray from '../../utils/parseKeyValueParamArray';
 import saveAssets from './saveAssets';
 import {logger} from '@react-native-community/cli-tools';
 import chalk from 'chalk';
+import fs from 'fs/promises';
```

Comment:
Please adjust this import to fix the Flow error.

<img width="920" alt="image" src="https://github.com/user-attachments/assets/cfe3155a-768a-4741-a3ae-ebdbde5c29a3">

```suggestion
import {promises as fs} from 'promises';
```